src/golang/lib/repos/sqlite/operator.go

```diff
@@ -93,6 +93,152 @@ const operatorNodeViewSubQuery = `
 	WHERE op_with_outputs.outputs IS NULL
 `
 
+const mergedNodeViewSubQuery = `
+	WITH
+		operator_node AS (
```

Comment:
If these `op_node` and `artf_node` are exactly the same as `GetOpNode` queries, we should make the shared part another constant and deduplicate using `var mergedNodeViewSubquery = fmt.Sprintf("... %s ...", sharedQuery)`.